src/odb/include/odb/db.h

```diff
@@ -7583,8 +7583,13 @@ class dbPowerSwitch : public dbObject
   void setOutSupplyPort(const std::string& out_port);
   void addControlPort(const std::string& control_port);
   void addOnState(const std::string& on_state);
+  void addLibCell(const std::string& lib_cell);
```

Comment:
this should take in a dbMaster

Reply:
This one was added for the upf commands, I've added another one that takes dbMaster

Reply:
updated